src/config-types.ts

```diff
@@ -27,8 +27,9 @@ export type Options = {
     max_width: number; // Maximum width of original images - if bigger => resized output
     external: {
       process:
-        | 'off' // Default
-        | 'download'; // Experimental
+      | 'off' // Default
+      | 'download' // Experimental
+      | 'cdn-resize'; // Even more experimental
```

Comment:
I think I would rather create a new `images.cdn` entry so we can manage external images and cdn images separately.

Reply:
Because you may want to enrich or download external images but treat cdn images as CDN.

```
{
  image: {
        cdn: {
            process: 'off' | 'autodetect',
            src_include: RegExp;
            src_exclude: RegExp | null;
            ...
        }
   }
}
```

Reply:
Make sense, will do